api/types/events/events.proto

```diff
@@ -131,6 +134,9 @@ message KubernetesPodMetadata {
 
 // SessionStart is a session start event
 message SessionStart {
+    reserved 11;
```

Comment:
I'm not familiar - what is this `reserved` magic?

Reply:
A `protoc`-enforced reminder to not use that field number or field name in the future; see https://developers.google.com/protocol-buffers/docs/proto3#reserved .

I felt it was appropriate as we're removing a field on what's technically a publicly used message type; perhaps I could move it down and leave a comment about it - the `access_requests` json tag comes from `UserMetadata` now.

Reply:
Gotcha, that makes sense. Have to be careful changing things in events.proto as some of those types are used in session recordings, and you dont want to break parsing an old session.

In this case you should be okay though as I don't think the session start event is ever part of the recorded session.

Reply:
I believe the whole point of reserving fields is that those fields in old versions of the message would just get ignored; this only applies for messages in protobuf form however - in json form the access requests would get parsed as part of `UserMetadata`, which doesn't even lose us any data.